src/components/tuits/tuit.js

```diff
@@ -47,14 +47,18 @@ const Tuit = ({tuit, deleteTuit, likeTuit, dislikeTuit}) => {
                     <i onClick={() => deleteTuit(tuit._id)}
                        className="fas fa-remove tuit-button fa-2x fa-pull-right"/>
                 }
-                <Link to={`/tuit/${tuit._id}`}>
+                { tuit.ownedByMe === true &&
+                    <Link to={`/tuit/${tuit._id}`}>
                     <i className="float-end tuit-button fas fa-circle-ellipsis me-1"/>
                 </Link>
-                <h2
-                    className="fs-5">
-                    {tuit.postedBy && tuit.postedBy.username}
-                    @{tuit.postedBy && tuit.postedBy.username} -<span className="ms-1">{daysOld(tuit)}</span> </h2>
-                {tuit.tuit}
+                }
+                <div>
+                <Link
+                    className="fs-5"
+                    to={`/other-profile/${tuit.postedBy._id}`}>
```

Comment:
Make sure that when a user clicks on the link to another user, and that user is itself, that it directs to their own profile instead of the other-user link